File: app/src/main/java/ru/p3tr0vich/fuel/SystemServicesHelper.java
package ru.p3tr0vich.fuel;

import android.accounts.AccountManager;
import android.app.NotificationManager;
import android.content.Context;
import android.net.ConnectivityManager;
import android.view.inputmethod.InputMethodManager;

class SystemServicesHelper {

    private SystemServicesHelper() {
    }

    public static NotificationManager getNotificationManager(Context context) {
        return (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
    }

    public static ConnectivityManager getConnectivityManager(Context context) {
        return (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
    }

    public static AccountManager getAccountManager(Context context) {
        return (AccountManager) context.getSystemService(Context.ACCOUNT_SERVICE);
    }

    public static InputMethodManager getInputMethodManager(Context context) {
        return (InputMethodManager) context.getSystemService(Context.INPUT_METHOD_SERVICE);
    }
}

File: app/src/main/kotlin/helpers/SystemServicesHelper.kt
package ru.p3tr0vich.fuel.helpers

import android.accounts.AccountManager
import android.app.NotificationManager
import android.content.Context
import android.net.ConnectivityManager
import android.view.inputmethod.InputMethodManager

object SystemServicesHelper {

    @JvmStatic
    fun getNotificationManager(context: Context?): NotificationManager? {
        return context?.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager?
    }

    @JvmStatic
    fun getConnectivityManager(context: Context?): ConnectivityManager? {
        return context?.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager?
    }

    @JvmStatic
    fun getAccountManager(context: Context?): AccountManager? {
        return context?.getSystemService(Context.ACCOUNT_SERVICE) as AccountManager?
    }

    @JvmStatic
    fun getInputMethodManager(context: Context?): InputMethodManager? {
        return context?.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager?
    }
}